src/main.css

```diff
@@ -1,3 +1,281 @@
+
+  dfn {
+    font-style: italic;
+  }
+
+  mark {
+    background-color: #ff0;
+    color: #000;
+  }
+
+  sub,
+  sup {
+    font-size: 75%;
+    line-height: 0;
+    position: relative;
+    vertical-align: baseline;
+  }
+
+  sub {
+    bottom: -0.25em;
+  }
+
+  sup {
+    top: -0.5em;
+  }
+
+  input {
+    border-radius: 0;
+  }
+
+  button,
+  [type="button"],
+  [type="reset"],
+  [type="submit"],
+  [role="button"] {
+    cursor: pointer;
+  }
+
+  [disabled] {
+    cursor: default;
+  }
+
+  [type="number"] {
+    width: auto;
+  }
+
+  [type="search"] {
+    -webkit-appearance: textfield;
+  }
+
+  [type="search"]::-webkit-search-cancel-button,
+  [type="search"]::-webkit-search-decoration {
+    -webkit-appearance: none;
+  }
+
+  textarea {
+    overflow: auto;
+    resize: vertical;
+  }
+
+  button,
+  input,
+  optgroup,
+  select,
+  textarea {
+    font: inherit;
+  }
+  optgroup {
+    font-weight: bold;
+  }
+
+  button {
+    overflow: visible;
+  }
+
+  button::-moz-focus-inner,
+  [type="button"]::-moz-focus-inner,
+  [type="reset"]::-moz-focus-inner,
+  [type="submit"]::-moz-focus-inner {
+    border-style: 0;
+    padding: 0;
+  }
+
+  button:-moz-focusring,
+  [type="button"]::-moz-focus-inner,
+  [type="reset"]::-moz-focus-inner,
+  [type="submit"]::-moz-focus-inner {
+    outline: 1px dotted ButtonText;
+  }
+
+  button,
+  html [type="button"],
+  [type="reset"],
+  [type="submit"] {
+    -webkit-appearance: button;
+  }
+
+  button,
+  select {
+    text-transform: none;
+  }
+
+  button,
+  input,
+  select,
+  textarea {
+    background-color: transparent;
+    border-style: none;
+    color: inherit;
+  }
+
+  select {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+  }
+
+  select::-ms-expand {
+    display: none;
+  }
+
+  select::-ms-value {
+    color: currentColor;
+  }
+
+  legend {
+    border: 0;
+    color: inherit;
+    display: table;
+    max-width: 100%;
+    white-space: normal;
+  }
+
+  ::-webkit-file-upload-button {
+    -webkit-appearance: button;
+    font: inherit;
+  }
+  [type="search"] {
+    -webkit-appearance: textfield;
+    outline-offset: -2px;
+  }
+
+  img {
+    border-style: none;
+  }
+  progress {
+    vertical-align: baseline;
+  }
+
+  svg:not(:root) {
+    overflow: hidden;
+  }
+
+  audio,
+  canvas,
+  progress,
+  video {
+    display: inline-block;
+  }
+
+  @media screen {
+    [hidden~="screen"] {
+      display: inherit;
+    }
+    [hidden~="screen"]:not(:active):not(:focus):not(:target) {
+      position: absolute !important;
+      clip: rect(0 0 0 0) !important;
+    }
+  }
+
+  [aria-busy="true"] {
+    cursor: progress;
+  }
+  [aria-controls] {
+    cursor: pointer;
+  }
+  [aria-disabled] {
+    cursor: default;
+  }
+
+  ::-moz-selection {
+    background-color: #b3d4fc;
+    color: #000;
+    text-shadow: none;
+  }
+
+  ::selection {
+    background-color: #b3d4fc;
+    color: #000;
+    text-shadow: none;
+  }
+
+  h1 {
+      color: green;
```

Comment:
Too much space :D

Reply:
Ugh... I'm sorry for this kind of mistake.